eessi_container.sh

```diff
@@ -428,6 +510,7 @@ fi
 [[ ${VERBOSE} -eq 1 ]] && echo "CONTAINER=${CONTAINER}"
 
 # set env vars and create directories for CernVM-FS
+# TODO need to use separate values for separate repos?
```

Comment:
I don't think that's really necessary, it's fine to have just one cache directory.

Reply:
```suggestion
```